freqtrade/freqtradebot.py

```diff
@@ -54,6 +56,7 @@ def __init__(self, config: Dict[str, Any])-> None:
         self.rpc: RPCManager = RPCManager(self)
         self.persistence = None
         self.exchange = Exchange(self.config)
+        self.edge = Edge(self.config, self.exchange)
```

Comment:
I think edge should only be instanciated when it's active... 
doesn't make sense if i'm not interrested in this functionality to instanciate and keep it in memory - while surely not too much - it's still another object we're not going to use